docs/src/pages.ts

```diff
@@ -14,6 +14,60 @@ const pages: readonly MuiPage[] = [
     },
   },
   { pathname: '/blog', title: 'Blog', icon: standardNavIcons.BookIcon },
```

Comment:
No supported anymore

```suggestion
  { pathname: '/blog', title: 'Blog' },
```

Reply:
I'll remove it, thanks for pointing it out!